Solutions/alexmarchis1990/p05_alexmarchis1990.swift

```diff
@@ -0,0 +1,18 @@
+import NinetyNineSwiftProblems
+
+extension List {
+    
+    func reverse() -> List<T> {
+        
+        var arr:[T] = []
+        arr.append(self.value)
+        var currentNode = self.nextItem
+        
+        while let cn = currentNode {
+            arr.insert(cn.value, at: 0)
+            currentNode = cn.nextItem
+        }
+        
+        return List<T>.init(arr)!
```

Comment:
Using `Array` or any other sequence types from the standard library is not allowed, as it defeats the purpose of these problems.